hazelcast-vnet/README.md

```diff
@@ -0,0 +1,14 @@
+# Deploy a Hazelcast Cluster
```

Comment:
Hmm, scale set VMs are also in the same VNET, in fact in the new stack you can't have a vm without a VNET. How about naming this hazelcast-vm-cluster and in a few months time hazelcast-vmss-cluster?